cmd/cloudx/eventstreams/create.go

```diff
@@ -0,0 +1,54 @@
+// Copyright © 2024 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package eventstreams
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ory/cli/cmd/cloudx/client"
+	cloud "github.com/ory/client-go"
+	"github.com/ory/x/cmdx"
+	"github.com/ory/x/flagx"
+)
+
+func NewCreateEventStreamCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "event-stream [--project=PROJECT_ID] --type=sns --aws-iam-role-arn=arn:aws:iam::123456789012:role/MyRole --aws-sns-topic-arn=arn:aws:sns:us-east-1:123456789012:MyTopic",
```

Comment:
If a flag is required, shouldn't it be a positional argument instead?

Reply:
Is that a rule? Considering you need to provide a bunch of info, that’d get complicated real quick.

Reply:
Not really a rule, just feels weird to have required flags in a CLI application. But I agree, the values are probably too convoluted to make them positional arguments.

Reply:
Just to weigh in with my anecdotal opinion, I like flags here because they are more documenting that just a huge list of positional arguments.